src/project/project.service.ts

```
@@ -31,6 +31,9 @@ import { User } from 'src/user/entities/user.entity';
 
 @Injectable()
 export class ProjectService {
+  getUserProjects(userId: string): any {
```

ESLint

[error] 'userId' is defined but never used.

(@typescript-eslint/no-unused-vars)